pkg/test/runner/runner.go

```diff
@@ -290,3 +305,34 @@ func newExpected(path string) (expected, error) {
 
 	return e, nil
 }
+
+func updateExpected(tmpPkgPath, resultsPath, sourceOfTruthPath string) error {
```

Comment:
While using e2e, I have realized the diff format is not super friendly for debugging or expressing the output as human. It is optimized for the machine :)
What have been your experience.

Reply:
I will create PR to improve this. Will merge this one.